shacl12-vocabularies/shacl-shacl.ttl

```diff
@@ -111,6 +111,7 @@ shsh:ShapeShape
 	sh:targetSubjectsOf sh:uniqueLang ;
 	sh:targetSubjectsOf sh:uniqueMembers ;
 	sh:targetSubjectsOf sh:xone ;
+  sh:targetSubjectsOf sh:singleLine ;
```

Comment:
Should alignment of this line not match those above? I still hate tabs for this purpose, but it doesn't make sense to indent one line with two-spaces when the rest of its stanza has one-tab indents.
```suggestion
	sh:targetSubjectsOf sh:singleLine ;
```